Raven.NET.Core/Observers/Interfaces/IRavenWatcher.cs

```diff
@@ -1,7 +1,14 @@
+using System;
+using Raven.NET.Core.Configuration;
+using Raven.NET.Core.Subjects;
+
 namespace Raven.NET.Core.Observers.Interfaces
 {
     public interface IRavenWatcher : IRaven
     {
-        
+        public IRavenWatcher Create(string name, Func<RavenSubject, bool> callback, Action<RavenSettings> options = null);
```

Comment:
Some documentation or inherit doc could be useful here. Just for reference

Reply:
good point